src/pages/quest/index.js

```diff
@@ -446,10 +446,14 @@ function Quest() {
             if (objective.shotType !== 'kill') {
                 verb = t('Shoot');
             }
+            let shootString = '{{shootOrKill}} {{target}} x {{count}}';
+            if (objective.count < 2) {
+                shootString = '{{shootOrKill}} {{target}}';
+            }
             taskDetails = (
                 <>
                     <>
-                        {t('{{shootOrKill}} {{target}} {{count}} times', {
+                        {t(shootString, {
```

Comment:
We used `t` here to translate `times` respecting _declensions_. In this new format, all components are already individually translated so it's not needed.

Reply:
Good point!